src/app/login/login.scss

```diff
@@ -0,0 +1,22 @@
+@import url(https://fonts.googleapis.com/css?family=Atomic+Age);
+#divCenter {
```

Comment:
dont use camelCases class name 
names are not matching ... its called divCenter but u r ading bottom padding